integration/testdata/6.1/config/application.rb

```diff
@@ -24,6 +24,8 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
 
+    # config.assets.prefix = "custom"
```

Comment:
Yeah, this one it is actually being used. 

This rails setting changes the path were the assets are placed after the `assets:precompile` command. By default, it will be `public/assets`, and this setting will change it to `public/custom`. It is commented because uncommenting this would break all other tests. So, during the [appropriate test case I remove this comment](https://github.com/doximity/rails-assets/blob/bernardolins-custom-assets-precompile/integration/reuse_layer_rebuild_test.go#L205). 

This is kinda hacky, but using this setting is an easy way to simulate an app that stores precompiled assets in different paths. 

I think it might be a better way to enable this without the need to keep the comment there. I will try to figure it out :)

Reply:
Ah, I hadn't made that connection.

Is it possible to append that line to the config during the test, rather than uncomment it?

If not, then I think we should add a comment to the config explaining why that line is commented out. I just worry that it will look like tech. debt and some day someone will delete it and be very confused why tests start failing.

Reply:
I've [changed the test](https://github.com/doximity/rails-assets/blob/bernardolins-custom-assets-precompile/integration/reuse_layer_rebuild_test.go#L200) to append a line to the `config/initializers/assets.rb` file instead, this should have the same effect without the need to keep that comment.